wiki/osu!tourney/Prizes/en.md

```diff
@@ -16,9 +16,29 @@ We may award profile badges to any **first-place winners** of community-run tour
 - The badge request is made **well before** the tournament commences play. (*We suggest at least two to three weeks.*)
 - Any of the staff members involved in managing the tournament are **not** participating in the tournament themselves.
 - The tournament is run four or fewer times per year (seasonal).
+- The list of players within a given tournament has been screened by the Support Team and verified to be free of any suspicious players.
 
 **¹**: "Experienced" team members are loosely defined as someone who has contributed significantly to the successful running of at least **three** badge-receiving tournaments beforehand, or has been a part of the osu! World Cup team.
 
+### Tournament Screening
+
+As part of awarding official support, promotion or other prizes, the Support Team offers a limited form of player vetting similar to the 'security checks' undergone by all World Cup players.
+
+**Only tournaments that qualify for all other badge criteria may apply for screening.** We highly suggest involving some experienced staff in your contest if you are planning to run a new tournament in order to qualify for this.
+
+Tournament organizers will be expected to provide the following:
+
+- A comma separated list of player usernames, user ids and the 2 digit ISO code of their profile country
```

Comment:
Is it possible to add an short example of what it should look like ? Like that there should be no problem with the format for anyone.